abci/client/client.go

```diff
@@ -31,6 +31,7 @@ type Client interface {
 	SetOptionAsync(types.RequestSetOption) *ReqRes
 	DeliverTxAsync(types.RequestDeliverTx) *ReqRes
 	CheckTxAsync(types.RequestCheckTx) *ReqRes
+	CheckQuorumSignatureAsync(types.RequestCheckQuorumSignature) *ReqRes
```

Comment:
We agreed to use the existing query connection and query method.

Reply:
As I was going through the code, I just can't bring myself to use the query method, it just doesn't seem clean enough to me and I couldn't find a way to make it feel clean. I switched to query connection though. If anyone feels really strongly about this they can write the code switch.

Reply:
No problem. I'll do PR tomorrow.